silverback/_cli.py

```diff
@@ -18,9 +18,10 @@ def cli():
 @verbosity_option()
 @network_option(default=None)
 @click.option("--account", type=AccountAliasPromptChoice(), default=None)
+@click.option("--runner", default=None)
```

Comment:
some guidance on how to set this (via `help=`) would be appreciated, same with some README update